okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -489,9 +502,9 @@ public void close() {}
 
         FrameWriter rawFrameWriter;
         synchronized (lock) {
-          socket = sock;
+          socket = Preconditions.checkNotNull(sock);
```

Comment:
fyi, we typically add a comment to checkNotNull to say which thing was null.  Its very helpful in user reported bugs since the line numbers may be off.

Reply:
Added comment